fragment_internal_test.go

```diff
@@ -2455,6 +2465,14 @@ func (f *fragment) mustSetBits(rowID uint64, columnIDs ...uint64) {
 	}
 }
 
+func addToBitmap(bm *roaring.Bitmap, rowID uint64, columnIDs ...uint64) {
+	for i, c := range columnIDs {
+		columnIDs[i] = rowID*ShardWidth + c%ShardWidth
+	}
+	positions := columnIDs
```

Comment:
i do not see why `positions` exists, at this point. is it okay that addToBitmap mutates the contents of the slice passed to it? (this will only matter in explicit `slice...` cases, i guess.)

Reply:
I was just trying to make it more clear that columnIDs no longer represented column ids and was now full of positions. Guess I should just have added a comment :)